samples/core/guide/autograph_control_flow.ipynb

```diff
@@ -0,0 +1,770 @@
+        "  # The result works like a regular op: takes tensors in, returns tensors.\n",
+        "  # You can inspect the graph using tf.get_default_graph().as_graph_def()\n",
+        "  input = tf.placeholder(tf.int32)\n",
+        "  result = fizzbuzz(input)\n",
+        "  with tf.Session() as sess:\n",
+        "    sess.run(result, feed_dict={input:10})    \n",
+        "    sess.run(result, feed_dict={input:11})    \n",
+        "    sess.run(result, feed_dict={input:12})    \n",
+        "    sess.run(result, feed_dict={input:13})    \n",
+        "    sess.run(result, feed_dict={input:14}) \n",
+        "    sess.run(result, feed_dict={input:15}) \n",
+        "    "
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "-pkEH6OecW7h",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Assert\n",
+        "\n",
+        "Let's try some other useful Python constructs, like `print` and `assert`. We automatically convert Python `assert` statements into the equivalent `tf.Assert` code.  "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "IAOgh62zCPZ4",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "def f(x):\n",
+        "  assert x != 0, 'Do not pass zero!'\n",
+        "  return x * x\n",
+        "\n",
+        "tf_f = autograph.to_graph(f)\n",
+        "\n",
+        "with tf.Graph().as_default():  \n",
+        "  with tf.Session():\n",
+        "    try:\n",
+        "      print(tf_f(tf.constant(0)).eval())\n",
+        "    except tf.errors.InvalidArgumentError as e:\n",
+        "      print('Got error message:\\n%s' % e.message)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "KRu8iIPBCQr5",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Print\n",
+        "\n",
+        "You can also use plain Python `print` functions in in-graph"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "ySTsuxnqCTQi",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "@autograph.convert()\n",
+        "def f(n):\n",
+        "  if n >= 0:\n",
+        "    while n < 5:\n",
+        "      n += 1\n",
+        "      print(n)\n",
+        "  return n\n",
+        "    \n",
+        "with tf.Graph().as_default():\n",
+        "  with tf.Session():\n",
+        "    f(tf.constant(0)).eval()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "NqF0GT-VCVFh",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Lists\n",
+        "\n",
+        "Appending to lists in loops also works (we create a `TensorArray` for you behind the scenes)"
```

Comment:
This is no longer always the case. Rephrasing to this would be ok: "(we create tensor list ops for you behind the scenes)"

Reply:
Done